pkg/daemon/server/service/pipeline_metrics_query.go

```diff
@@ -196,15 +214,17 @@ func (ps *pipelineMetadataQuery) GetVertexMetrics(ctx context.Context, req *daem
 				return nil, err
 			}
 
-			// Check if the resultant metrics list contains the processingRate, if it does look for the period label
-			if value, ok := result[metrics.VertexProcessingRate]; ok {
-				metricsList := value.GetMetric()
-				for _, metric := range metricsList {
-					labels := metric.GetLabel()
-					for _, label := range labels {
-						if label.GetName() == metrics.LabelPeriod {
-							lookback := label.GetValue()
-							processingRates[lookback] = metric.Gauge.GetValue()
+			if !ps.useNewRateCalculation {
```

Comment:
will this part of the code be deleted once we make `useNewRateCalculation` the standard? if yes, could you please add a TODO?